project.clj

```diff
@@ -10,18 +10,23 @@
                  [turbovote.datomic-toolbox "0.2.5-SNAPSHOT" :exclusions [com.datomic/datomic-pro]]
                  [com.datomic/datomic-pro "0.9.5130" :exclusions [org.slf4j/slf4j-nop]]
                  [clj-aws-s3 "0.3.8"]
+                 [clj-time "0.9.0"]
                  [democracyworks.squishy "1.0.0"]
                  [org.clojure/data.csv "0.1.2"]
-                 [turbovote.imbarcode "0.1.4-SNAPSHOT"]
+                 [turbovote.imbarcode "0.1.4-SNAPSHOT"
```

Comment:
We should really not depend on snapshots. Does this actually need unreleased functionality and can we release it if so? @tie-rack?